newclient/src/modules/blocks/handlers/BlockStats/BlockStats.vue

```diff
@@ -130,8 +127,7 @@ export default class BlockStats extends Mixins(NumberFormatMixin) {
     */
     @Watch('newBlock')
     onNewBlockChanged(newVal: number, oldVal: number): void {
-        if (newVal && newVal != oldVal) {
-            this.timestamp = new Date().toString()
+        if (newVal && newVal != oldVal && newVal > this.initialLoadedBlock) {
```

Comment:
I used initialLoadedBlock instead of latestBlockNumber since that just changes it to a string